Tests/LibWeb/Text/expected/wpt-import/css/cssom/serialize-values.txt

```diff
@@ -2,8 +2,8 @@ Harness status: OK
 
 Found 687 tests
 
-658 Pass
-29 Fail
+659 Pass
```

Comment:
This commit should be squashed into the previous commit. We are big fans of atomic commits in this project - every commit should build and pass tests on its own. If we were to bisect between this commit and the previous one, this test would fail.

(More practically, if we had to revert the previous commit, we would then also have to know to revert this one).